### PR TITLE
RAC-5369 Document the immediate rediscovery workflow

### PR DESCRIPTION
**Pull request by derrickostertag**
This covers both RAC-5369 and RAC-5371
* Document the immediate rediscovery workflow
* Document the delayed rediscovery workflow
@manfrednde 
@jimturnquist

### CODE REVIEW

**Comment by manfrednde** [comment]
:+1:

File: docs/rackhd/rediscovery.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                  "nodeId": "<nodeId>"
+              }
+            }' \
+        <server>/api/2.0/workflows
+
+
+Delayed Refresh Node Discovery
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An user can defer a node discovery by posting to /api/2.0/workflows with a payload. The user will need to manually reboot the node after executing the API before the node re-discovery/refresh process can start.
```
Comment (jimturnquist) [inline docs/rackhd/rediscovery.rst]: A user

File: docs/rackhd/rediscovery.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                      "nodeId": "<nodeId>"
+                  },
+                  "nodeId": "<nodeId>"
+              }
+            }' \
+        <server>/api/2.0/workflows
+
+**Manually rebooting the node using ipmitool example**
+
+.. code-block:: REST
```
Comment (jimturnquist) [inline docs/rackhd/rediscovery.rst]: Should this example be in the form of a workflow invocation instead of a direct ipmi call?

Comment (derrickostertag) [inline docs/rackhd/rediscovery.rst]: No. You will get an "unable to run multiple workflows on a single node" error.

**Pull request merged**